app/models/user.rb

```diff
@@ -14,6 +14,16 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  # Returns the hash digest of the given string.
+  def User.digest string
+    cost = if ActiveModel :: SecurePassword.min_cost
```

Comment:
Chỗ này có thể dùng toán tử 3 ngôi được không nhỉ? Nếu dùng được thì ông đổi sang dùng toán tử 3 ngôi cho gọn nhé

Reply:
:v Theo sách ý mà, để tui chuyển